tests/test_workers/test_tasks/test_build_create_empty_index.py

```diff
@@ -44,7 +44,9 @@ def test_get_no_present_operators(mock_grpcurl, tmpdir):
 @mock.patch('iib.workers.tasks.build_create_empty_index._push_image')
 @mock.patch('iib.workers.tasks.build_create_empty_index._create_and_push_manifest_list')
 @mock.patch('iib.workers.tasks.build_create_empty_index._update_index_image_pull_spec')
+@mock.patch('iib.workers.tasks.build_create_empty_index.is_image_dc')
 def test_handle_create_empty_index_request(
```

Comment:
you should also test with mock_iidc.return_value = True and tests error is raised